packages/react-chat/src/components/Members/MembersList.tsx

```diff
@@ -27,13 +27,26 @@ export function MembersList() {
     }
   }, [contacts, identity]);
 
+  const members = useMemo(
+    () =>
+      activeChannel &&
+      activeChannel?.type === "group" &&
+      activeChannel.members &&
+      identity
+        ? activeChannel.members.filter(
+            (e) => e.id !== utils.bufToHex(identity.publicKey)
+          )
+        : userContacts,
+    [activeChannel]
+  );
+
   const onlineContacts = useMemo(
-    () => userContacts.filter((e) => e.online),
-    [userContacts]
+    () => members.filter((e) => e.online),
+    [members]
```

Comment:
Group members are always offline  because there is no `isOnline` property  in  GroupMember type `groupChats.d.ts`

Reply:
Will rework that later